### ISSUE TITLE
Generate per-workspace CER + WER

### ISSUE THREAD

**Issue by mikegerber** [qurator-spk/dinglehopper#35]
1. It's easy to calculate this from the individual CER/WER and the character/word counts.
2. But how to save a global JSON report in the METS? It would not "manifest a physical page" which OCR-D seems to demand for any file

**Comment by mikegerber** [comment]
>     2. But how to a save a global JSON report in the METS? It would not "manifest a physical page" which OCR-D seems to demand for any file

@kba @bertsky @cneud Any thoughts on this?

**Comment by bertsky** [comment]
> > But how to save a global JSON report in the METS? It would not "manifest a physical page" which OCR-D seems to demand for any file
> 
> @kba @bertsky @cneud Any thoughts on this?

Yes, this became possible when we [agreed on](https://github.com/OCR-D/spec/pull/154) an "official" way to have global (document-wide) files in the METS. You just put it in there without a pageId (i.e. without a reference of the file in the structMap) and use a certain convention for the file ID (with `FULLDOWNLOAD` IIRC).

Now that different MIME types are allowed in fileGrps, having an output fileGrp with page-wise and document-global reports should be no problem.

**Comment by bertsky** [comment]
BTW I believe having a measurement of CER standard deviation or variance is also useful. See [here](https://github.com/ASVLeipzig/cor-asv-ann/blob/0ae6867eba39f73f5832b219f09f71788145d1c2/ocrd_cor_asv_ann/lib/alignment.py#L414-L433) for an implementation.

**Comment by mikegerber** [comment]
> BTW I believe having a measurement of CER standard deviation or variance is also useful. See [here](https://github.com/ASVLeipzig/cor-asv-ann/blob/0ae6867eba39f73f5832b219f09f71788145d1c2/ocrd_cor_asv_ann/lib/alignment.py#L414-L433) for an implementation.

True, I just want to give this (and any mean calculations) more thought, because we deal with rates

**Comment by mikegerber** [comment]
(Closing the issue was an accident, I often hit the wrong buttons)
